added create-book.feature

sachinb7 opened the pull request:
Positive Test - Create Book (Admin)
Create a new book with valid title and author using admin credentials
Verify the book is created successfully with status code 201
Check that the returned book object matches the input data

sachinb7 committed:
Assigned - API TEST CASES - Member 2 - 204184B

sachinb7 committed:
Merge branch 'main' of https://github.com/UOM-B20/is3440-itqa-assignment into 204184B

sachinb7 committed:
added create_book_admin.steps.js

sachinb7 committed:
updated

kingmalitha commented:
@sachinb7  please update according to new test case scenario and new folder structure thanks

sachinb7 committed:
Merge branch 'main' of https://github.com/UOM-B20/is3440-itqa-assignment into 204184B

sachinb7 committed:
updated with major changes

- removed create_book_admin.steps.js file
- added create-book.feature file
- updated book.step.js file

sachinb7 committed:
Update create-book.feature

sachinb7 committed:
Merge branch 'main' into 204184B

sachinb7 committed:
Merge branch 'main' of https://github.com/UOM-B20/is3440-itqa-assignment into 204184B

sachinb7 committed:
Merge branch '204184B' of https://github.com/UOM-B20/is3440-itqa-assignment into 204184B

sachinb7 commented:
@kingmalitha  updated accordingly

kingmalitha committed:
Merge branch 'main' into 204184B

kingmalitha reviewed (approved):
lgtm
